cmd/juju/status/status_internal_test.go

```diff
@@ -5338,9 +5338,11 @@ func (s *StatusSuite) TestFormatTabularCAASModelTruncatedVersion(c *gc.C) {
 				},
 			},
 			"bar": {
+				Charm: "bar",
+				CharmOrigin: "charmhub",
 				Scale:   1,
 				Address: "54.32.1.3",
-				Version: "registry.jujucharms.com/mine/image:0.5",
```

Comment:
these URLs are actually incomplete as it is 'registry.jujucharms.com/user/charm/resource'
I don't think it matters for testing purposes, but it would be good to have correct URLs.

Reply:
> I don't think we should have `cs:` or `ch:`. `ch` is an internal representation that we shouldn't leak outside of juju. I know that it does work when deploying, but in reality, I do wonder if we should prevent that.
> 
> I'd rather just drop the prefix and let people use `Store` to find out where the image is from.

The reason for doing it was to signal that the displayed path comes from the resource definition inside a charm rather than a oci image path from a different registry. Without the "ch:", the user sees just "mysql_image" which is the resource name defined in charm metadata and is pretty meaningless without the hint as to where it comes from. Compare this to displaying "mariadb" which is reasonably interpreted as an upstream oci image name.

Reply:
As per another comment, "res:" is used to indicate that the image path is a charm resource.
Test also updated to use a correct jujucharms url